### ISSUE TITLE
(Suggestion) Low friction minecarts

### ISSUE THREAD

**Issue by santibag** [CovertJaguar/Railcraft#618]
Empty minecart are slowing down immediately after getting boosted. That's useful when operating minecarts without a train. When a minecart run, it doesn't go a long distance so it's easy to catch. But when we use them as a train carriage, they prevent train to go high speed. After passing a set of h.s. booster (or transition) tracks, train slows down immediately. My suggestion is adding train carts which doesn't slow down a train after a boost when empty. Another option is disabling this slowing down feature from regular minecarts. But it will cause some running minecart trouble for single cart operators. Train carts are slightly different from regular minecarts with a crafting recipe difference and they cannot be moved by simply pressing forward key. That's because they're train carts made for trains. Another option is just adding a low friction minecart which can be used as a regular minecart and can go as long as a minecart with rider.
I want to make passenger trains but thay cannot go at high speed.

**Comment by CovertJaguar** [comment]
Linked Trains have higher drag, this can be disabled in the config.

All minecarts are given the same drag values, overriding the vanilla values. Assuming the mechanisms for replacing the vanilla instances are working, Cauldron needs an update for this.

**Comment by PikadudeNo1** [comment]
> Linked Trains have higher drag, this can be disabled in the config.

It can? The only relevant config option I see is for reverting all carts, linked or not, to vanilla drag. Having an option specifically to disable the extra train drag would be nice; if you like, I'll file a separate issue for that.

**Comment by CovertJaguar** [comment]
I believe it requires disabling the Locomotive Module.

**Comment by santibag** [comment]
Today I made some experiments on a high speed line. I placed 10 unpowered hs trans tracks to slow down returning train+ 10 powered hs trans tracks to make train go hs. distance between unpowered tracks is 1 meter(it was the first start configuration). Then I placed a hs booster every 30 meters(my standard rail booster distance for loaded minecarts) At the end, I used the first start configuration to make carts slow down and return at high speed. These are results
There are 2 steam locos. One at the beginning and one at the last.

1: 10 unloaded minecarts
Result: Train never even reached to the running speed of steve.

2: 10 chest carts
Result: Train never slowed down from hs.

3: 11 tank carts (yeah 11 was an accident but 1 cart is not so much important)
Result: Amazingly the train couldn't reach high speed first but reached when returning.

4: 10 unloaded minecarts (after setting "adjustdrag" to false at config)
Result: Same with 3. I think I made a mistake for one direction so it caused slowing. But return was good. Carts reached to high speed speed succesfully.

Overall result: Setting " adjustdrag" to false solves problem. I made these experiments but I still have a doubt about it. I want to build a train station where 10 car trains are operated automatically with a timetable like real train stations. I think I will give you another suggestion about timetables or something like that. Warm regards.
